JASP-Engine/JASP/R/descriptives.R

```diff
@@ -1055,6 +1078,56 @@ Descriptives <- function(jaspResults, dataset, options) {
   return(p)
 }
 
+
+.descriptivesDotPlots <- function(dataset, options, variable){
+  
+  
+  if (options$splitby != "" ) {
+    # return a collection
+    split <- names(dataset)
+    
+    plotResult <- createJaspContainer(title=variable)
+    plotResult$dependOn(options="splitby", optionContainsValue=list(variables=variable))
```

Comment:
```suggestion
    plotResult$dependOn(optionContainsValue=list(variables=variable))
```
since we have `jaspResults[["DotPlots"]]$dependOn(c("splitby", "descriptivesDotPlot"))`, this already depends on `"splitby"`.